src/main.js

```diff
@@ -0,0 +1,51 @@
+import { createTripInfoTemplate } from './view/info-view.js';
+import { createNavigationTemplate } from './view/navigation-view.js';
+import { createFiltersTemplate } from './view/filters-view.js';
+import { createSortingTemplate } from './view/sorting-view.js';
+import { createListPointsTemplate } from './view/list-points-view.js';
+import { createEditingPointTemplate } from './view/point-edit-view.js';
+import { createNewPointTemplate } from './view/point-create-view.js';
+import { createShowPointTemplate } from './view/point-view.js';
+import { createStatisticsTemplate } from './view/stats-view.js';
+import { createMessagesTemplate } from './view/messages-view.js';
+
+const POINT_COUNT = 3;
+
+const RenderingLocation = {
+  BEFOREBEGIN: 'beforebegin',
+  AFTERBEGIN: 'afterbegin',
+  BEFOREEND: 'beforeend',
+  AFTEREND: 'afterend'
+};
+
+const renderTemplate = (container, template, position) => {
```

Comment:
```suggestion
const renderTemplate = (container, template, position = RenderingLocation. BEFORE_END) => {
```

Можно задать значение параметра по умолчанию, и тем самым не передавать этот параметр, когда его значение соответствует BEFORE_END